cmd/init-repo/main.go

```diff
@@ -68,11 +70,21 @@ func main() {
 	if *repoOrg != "" {
 		cfg.SourceOrg = *repoOrg
 	}
+	if *githubHost != "" {
+		cfg.GithubHost = *githubHost
+	}
+	if *packageName != "" {
+		cfg.Package = *packageName
```

Comment:
this logic is strange. Shouldn't we preserve this if set, i.e. skip the following if clause?

Reply:
Oh! That's a good catch. The following `if` should only be run when `cfg.Package == ""`.